eval.go

```diff
@@ -104,6 +104,14 @@ func (e *setExpr) eval(app *app, args []string) {
 			app.ui.sort()
 			app.ui.loadFile(app, true)
 		}
+	case "globfilter", "noglobfilter", "globfilter!":
+		err = applyBoolOpt(&gOpts.globfilter, e)
+		if err == nil {
+			app.nav.sort()
+			app.nav.position()
+			app.ui.sort()
+			app.ui.loadFile(app, true)
+		}
```

Comment:
These lines of code are only needed for options which change the list of files shown, in order to refresh the UI - for example `hidden` or anything used by `filter`.

Having these lines for `globfilter` is correct (what you have done here), but I think these lines are no longer needed for `globsearch` (no longer used by `filter`) and can be removed.